### PR TITLE
USHIFT-2806: Initial implementation of RHEL 9.4 bootc test

### PR DESCRIPTION
**Pull request by ggiguash**

### CODE REVIEW

File: test/bin/pyutils/common.py [hunk 1]
```diff
@@ -24,12 +24,15 @@ def print_msg(msg: str, file=sys.stderr):
...
 
 
-def get_env_var(var_name: str):
+def get_env_var(var_name: str, def_val: str = None):
     """Get an environment variable or exit if not set."""
     value = os.environ.get(var_name)
     if value is None:
-        print_msg(f"Error: {var_name} environment variable not set.")
-        sys.exit(1)
+        if def_val is None:
```
Comment (pmtk) [inline test/bin/pyutils/common.py:31]: What do you think about reducing nesting?
```
    if value:
        return value

    if def_val:
        return def_value

    print_msg(f"Error: {var_name} environment variable not set.")
    sys.exit(1)
```

File: test/bin/pyutils/build_bootc_images.py [hunk 2]
```diff
@@ -157,9 +159,16 @@ def process_containerfiles(groupdir, dry_run=False):
                 continue
 
         common.print_msg(f"Processing {cf_path}")
-        common.run_command(
-            ["podman", "build", "-t", cf_outname, "-f", cf_path,
-                os.path.join(IMAGEDIR, "rpm-repos")], dry_run)
+        # Run the container build command
+        build_args = [
+            "podman", "build",
+            "--authfile", PULL_SECRET,
```
Comment (pmtk) [inline test/bin/pyutils/build_bootc_images.py:165]: FYI: Alternative would be to copy the pull secret(s) to ~/.docker/config.json, but we already expect ~/.pull-secret.json in other places (e.g. rebase.sh) so I guess it's fine

**Review by pmtk — COMMENTED** [review]
More questions than requests, looks pretty good

File: test/bin/pyutils/build_bootc_images.py (hunk 2 above)
Comment (ggiguash) [reply test/bin/pyutils/build_bootc_images.py:165]: I'd prefer using the pull secret directly, not to mess up the user configuration files.

**Pull request merged**